docs/source/examples/supplychain/supplychain_transaction_family.rst

```diff
@@ -1,39 +1,39 @@
-***************************************
+*******************************
 Supply Chain Transaction Family
-***************************************
+*******************************
 
 Overview
 =========
 
-The Supplychain Transaction Processor (SCTP) provides a mechanism for
+The Supply Chain Transaction Processor (SCTP) provides a mechanism for
 storing records of asset ownership. The SCTP tracks records, which correlate
-to a real world item. Every Record has an owner and a custodaian, which can
+to a real world item. Every Record has an owner and a custodian, which can
 be the same Agent. The owner is the Agent the legally owns the good. The
-custodian is the Agent that is in possesion of the good, which may be a
+custodian is the Agent that is in possession of the good, which may be a
 warehousing service, transportation service, etc. Agents are actors in the
-system that are can manupulate the Records by updating them or transfering
+system that can manipulate the Records by updating them or transferring
 their role to other Agents. Agents are identified by their public key and
 authenticated by the signature on the transactions they submit. Transfer of
-roles (Owner, Custodian) betweeen Agents is done via an Application process,
-Where the Agent wishing to assume the role Applies for the new role and the
-Agent in the role currently can either accept or reject that applicaiton.
+roles (Owner, Custodian) between Agents is done via an Application process,
+where the Agent wishing to assume the role applies for the new role and the
+Agent in the role currently can either accept or reject that Application.
 
-This is an proof of concept Transaction Processor that is actively being
-developed. As such it has not been thoughly tested and there will be
-protentially breaking changes made to it.
+This is a proof of concept Transaction Processor that is actively being
+developed. As such it has not been thoroughly tested and there will potentially
+be breaking changes made to it.
```

Comment:
I am keeping the scope for this document strictly to correctly spelling and grammar errors. If I start rewriting it, that quickly becomes a JIRA Task.